google-oauth-client-bom/pom.xml

```diff
@@ -107,12 +107,14 @@
       </plugin>
       <plugin>
         <artifactId>maven-javadoc-plugin</artifactId>
+        <version>3.2.0</version>
         <configuration>
           <skip>true</skip>
         </configuration>
       </plugin>
       <plugin>
         <artifactId>maven-site-plugin</artifactId>
+        <version>3.9.0</version>
```

Comment:
Looks like a downgrade. We're on 3.9.1 from the plauginmanagement in the parent.